cli/uninstall/uninstall.go

```diff
@@ -226,25 +228,34 @@ func searchLatestVersion(linkName, binDst, headerDst string) (string, error) {
 		if util.Find(programsToSearch, programName) == -1 {
 			continue
 		}
+		isRightFormat, _ := util.IsValidCommitHash(matches["ver"])
 
-		ver, err := version.Parse(matches["ver"])
-		if err != nil {
-			continue
-		}
-		if strings.Contains(programName, "tarantool") {
-			// Check for headers.
-			if _, err := os.Stat(filepath.Join(headerDst, binaryName)); os.IsNotExist(err) {
+		if isRightFormat && !hashFound {
+			hashFound = true
```

Comment:
Please, check the existence of `include` directory for tarantool program.

Reply:
If there is no such test already, let's add a unit test for searching the latest version if one version/hash has headers directory and another one - doesn't.

Reply:
fixed